src/pages/404.tsx

```diff
@@ -0,0 +1,40 @@
+import Link from "next/link";
+
+export default function FourOhFour() {
+  return (
+    <main>
+      <div>
+        <div className="h-[calc(100vh-200px)] py-12 sm:py-8 md:py-12 lg:py-14 xl:py-12 2xl:py-28">
+          <div className="flex flex-col items-center">
+            <div className="max-w-md px-4 sm:max-w-2xl sm:px-6 md:max-w-3xl lg:max-w-4xl lg:px-0 xl:max-w-5xl 2xl:max-w-6xl">
+              <h1 className="text-center text-4xl font-bold tracking-tight text-white sm:text-6xl sm:tracking-tight lg:text-[4rem] xl:text-[6rem] xl:tracking-tight 2xl:text-[6.5rem]">
```

Comment:
```suggestion
              <h1 className="text-center text-4xl font-bold tracking-tight sm:text-6xl sm:tracking-tight lg:text-[4rem] xl:text-[6rem] xl:tracking-tight 2xl:text-[6.5rem]">
```

Reply:
This cannot be implemented as black text isn't visible on black bg

Reply:
@Nishith-Savla the bg ain't black tho

Running this branch in my PC is giving this output

![image](https://user-images.githubusercontent.com/89210438/212669151-28f4f6ef-3cf3-4e6e-b694-ac5e81487f78.png)

![image](https://user-images.githubusercontent.com/89210438/212669188-a8bafcb7-6634-461e-9627-ec637176a0a4.png)

The global.css file will apply the base styles on all pages.

Reply:
Ohh, got it. I didn't have the changes from the dev branch